mcworldlib/anvil.py

```diff
@@ -394,6 +407,54 @@ def uncache(self, pos: u.RegionPos, recursive: bool = False):
         self._items[pos] = candidates[0]
 
 
+def compress_lz4_java(data: bytes, compression_level: int = 6) -> bytes:
```

Comment:
Maybe make this magic 6 a top-module constant and use the constant as the default value?

```python
DEFAULT_COMPRESSION = 0x10  # Used by vanilla Minecraft, also the default in LZ4-Java
...
def compress_lz4_java(data: bytes, compression_level: int = DEFAULT_COMPRESSION) -> bytes:
```